server/tests/chat/test_tool_output_cap.py

```
@@ -219,3 +221,68 @@ def test_summarize_band_is_wide_enough_for_typical_outputs(self):
+        with pytest.raises(ValueError, match="must be a positive integer"):
+            _positive_int_env("AURORA_TEST_LIMIT", 40_000)
+
+    def test_missing_override_uses_default(self, monkeypatch):
+        monkeypatch.delenv("AURORA_TEST_LIMIT", raising=False)
+
+        assert _positive_int_env("AURORA_TEST_LIMIT", 40_000) == 40_000
+
+    def test_module_reads_environment_overrides_at_startup(self):
+        env = os.environ.copy()
+        env.update(
+            TOOL_OUTPUT_PASS_THROUGH_CHARS="16000",
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to argument: "TOOL_OUTPUT_PASS_THROUGH_CHARS"

(S106)